openedx/frontend-app-gradebook
# feat: Bulk Management reorganization, phase 1

**TL;DR -**
Re-organize the Bulk Management controls, remove tabs, convert Bulk Management tab to Bulk Management History View, and add an import success toast.

JIRA: [AU-43](https://openedx.atlassian.net/browse/AU-43)
figma: [figma] (https://www.figma.com/file/LHr3WTCMBe7GOm6MqXZwpk/Grading-Enhancements?node-id=329%3A125)

**What changed?**
Updated redux logic to:
* track and control active UI view, between grades and activity log
* Add trigger for showing an import success toast
Move "Tabs" to "Views"
* GradesTab => GradesView
* BulkManagementTab => BulkManagementHistoryView
Simplify BulkManagementHistoryView
* remove download button
* replace hints with help text (with new responsive behavior from Kevin)
* update header
Group BulkManagementControls
* BulkManagementControls now contains import and download grades buttons
* (20px margin between)
Other small updates to make this easier
* Add new InterventionsReport component to wrap the interventions download button and help text
* Split up Search Controls to remove FilterToggleButton (for cleaner view organization)
* Move messages to per-component files (was getting messy)
* Update margins around step display
* Rename UserLabel to FilteredUsersLabel and move it on the view
* updated test plan

**Developer Checklist**

**Testing Instructions** (updates made to test plan)
* Navigate between views (verify both tables still load and the help texts are updated)
* verify import grades works and brings up the Toast (and that the toast dismisses after 5 secs)
* Verify Download buttons (grades and interventions) still work
* smoke test

![image](https://user-images.githubusercontent.com/5533134/127604356-a7cdfac8-4363-4eda-81c9-8d07cc992c67.png)
![image](https://user-images.githubusercontent.com/5533134/127604401-01f1ca21-28e1-4712-aae2-c80e9c9e0268.png)
![image](https://user-images.githubusercontent.com/5533134/127604429-88490e3f-66ca-4856-a8c6-7bd72e9539f1.png)


**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
